### ISSUE TITLE
NESO-Tokamak org project board issue

### ISSUE THREAD

**Issue by jwscook** [ExCALIBUR-NEPTUNE/NESO-Tokamak#2]

**Comment by jwscook** [comment]
JE to meet RF to discuss next set of equations and BCs.

**Comment by jwscook** [comment]
Stefan needs ~1.5 weeks before we can lay the groundwork for adding reactions to this repo in a way that means we won't have to duplicate work

**Comment by jwscook** [comment]
Sheath boundary condition implemented from Hermes3. Q: what is purpose of slope limiters. A: cope with initial transients and lots of ringing in general.

Potential is cast to CG for this solve but maybe needs to be pushed back to DG for the BC.

Question for @mdave: does Nektar++ have slope limiters?

**Comment by jwscook** [comment]
Stefan is going to send around the notes on the basic equations for physical neutral processes.

**Comment by jwscook** [comment]
- [x] Stefan's document is hosted on the NESO-Tokamak github actions.
- [ ] Single fluid (1 week) 13th Nov
- [ ] Two variable incarnation (1 week) 20th Nov
- [ ] Add particles (4 weeks) 18th Dec
- [ ] Run it! (2 weeks at least) Jan...
- [ ] Nice-to-have: turbulence. Feb..

**Comment by jwscook** [comment]
We will need project and evaluate on a particle subgroup in NESO, as noted by @will-saunders-ukaea

**Comment by jwscook** [comment]
We'll need:

- [ ] ionisation
- [ ] recombination
- [ ] charge-exchange
- [ ] recycling (can be 100% specular reflection of neutrals)

**Comment by jwscook** [comment]
Working on changes to NESO to allow multiple species and reactions to be configured via xml.

**Comment by jwscook** [comment]
Maybe 1 more week until https://github.com/ExCALIBUR-NEPTUNE/NESO/pull/274 is ready to merge

**Comment by jwscook** [comment]
Recombination is not ready yet in the Reactions framework.
Wall-Reactions will be implemented next FY.
